pkg/cloud/utils.go

```diff
@@ -19,30 +19,32 @@ package cloud
 import (
 	"encoding/json"
 	"fmt"
+	"regexp"
 	"strings"
 
 	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/meta"
 )
 
 var (
-	gaPrefix    = "https://www.googleapis.com/compute/v1"
-	alphaPrefix = "https://www.googleapis.com/compute/alpha"
-	betaPrefix  = "https://www.googleapis.com/compute/beta"
+	computePrefix         = "https://www.googleapis.com/compute"
+	networkServicesPrefix = "https://www.googleapis.com/networkservices"
+	apiGroupRegex         = regexp.MustCompile(`([a-z0-1]*)\/([a-z0-1]*)?\/projects`)
```

Comment:
can we put this closer to the use and document what this is supposed to match

Reply:
This is in the same file it's used in, I think including this in the function itself would add a non-trivial performance cost, do you have any other recommendations?

Reply:
I meant move the definition right above the function that is using it. You can keep at as a static variable.

Reply:
Updated, thanks1